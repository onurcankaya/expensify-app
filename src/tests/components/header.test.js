import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import ReactShallowRenderer from 'react-test-renderer/shallow'
import Header from '../../components/header'

test('should render Header correctly', () => {
  const wrapper = shallow(<Header />)
  expect(toJSON(wrapper)).toMatchSnapshot()

  // expect(wrapper.find('h1').length).toBe(1)
  // const renderer = new ReactShallowRenderer()
  // renderer.render(<Header />)
  // expect(renderer.getRenderOutput()).toMatchSnapshot()
})
