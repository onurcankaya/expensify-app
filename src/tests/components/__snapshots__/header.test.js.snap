// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    Create Expense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/edit/:id"
  >
    Edit Expense
  </NavLink>
</header>
`;
